wysiwyg-rails.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |gem|
   gem.version       = WYSIWYG::Rails::VERSION
 
   gem.add_dependency "railties", ">= 3.2", "< 5.0"
-  gem.add_dependency "font-awesome-rails", "= 4.2.0.0"
+  gem.add_dependency "font-awesome-rails", "~> 4.3.0.0"
```

Comment:
Well as you said it should be fixed in a upcoming release, should I fix this myself or just wait for your release ?